### ISSUE TITLE
Implement continuous integration

### ISSUE THREAD

**Issue by robinpaulson** [robinpaulson/SimpleRT#62]
We can use this to test changes as we make them, also to build binary packages for Debian, Android, macOS and later for Windows.

It is built into Github, via "Actions", which includes a set of ready-to-go recipes for various build methods

**Comment by BraINstinct0** [comment]
going further, this Github Actions can be used to perform tasks other than build(e.g.tests, publish to Chocolatey/winget/etc) on specific triggers(e.g. new pr, new release, chaining actions, etc.)

**Comment by robinpaulson** [comment]
Yes, github actions are remarkably powerful, but I've struggled to get my head round them in the past. I don't know anything of those publishing targets, my focus first is for testing on new commits, secondly for binary distribution on gh. Maybe an apt repo third. Windoze is well down the list; we don't even have compliant code for that platform yet.

If you wish to put together an Action, go for it.

**Comment by BraINstinct0** [comment]
Sadly neither do I have much experience; I think publishing can be discussed after the build/test becomes stable. Before then I'll look at other repos and try to figure out some easy, reliable methods.(No guarantee, since I'm short of time for now)

**Comment by robinpaulson** [comment]
Publishing can be discussed at any time.

**Comment by robinpaulson** [comment]
CI for Android implemented
